src/main/java/com/google/devtools/build/lib/bazel/coverage/CoverageReportActionBuilder.java

```diff
@@ -248,6 +248,7 @@ private CoverageReportAction generateCoverageReportAction(
     ImmutableList<Artifact> inputs = ImmutableList.<Artifact>builder()
         .addAll(args.coverageArtifacts())
         .add(reportGeneratorExec)
+        .add(args.reportGenerator().getRunfilesSupport().getRunfilesMiddleman())
```

Comment:
Actually this may be null, can you null-check before adding it to `inputs`?

Reply:
Sure! Coming right up. Hopefully I'm using correct style/idioms. Thanks!